agent/bench-scripts/unittests

```diff
@@ -75,6 +91,8 @@ PATH=$(remove_path /opt/pbench-agent/util-scripts ${PATH})
 export PATH=${_testopt}/unittest-scripts:${_testopt}/util-scripts:${_testopt}/bench-scripts:${PATH}
 export PYTHONPATH=${_testopt}/common/lib:${PYTHONPATH}
 
+_install_venv
```

Comment:
Can we consider moving this out of the normal unit test script path and into a script designed for TravisCI so that we don't have to path the install price every time we run the unit tests?

It seems that we'll have virtual environments we'll be working in normally.  But not all of us currently work in virtual environments.